### PR TITLE
zPermissions Support

### PR DESCRIPTION
**Pull request by ctooley21**
Added zPermissions Support

### CODE REVIEW

**Comment by guipenedo** [comment]
Looks great! Just two things: could you use java 7, as many servers still aren't running java 8, and why did you change permissionsex database queries?

**Comment by ctooley21** [comment]
Sure thing in regards to Java 7. But in regards to pex database queries, I must have accidently been editing the wrong file when trying to work out how different zPermissions stores everything compared to Pex. We can revert those changes.

File: src/me/philipsnostrum/bungeepexbridge/modules/PermPlayer.java [hunk 1]
```diff
@@ -34,8 +35,9 @@ public PermPlayer (UUID uuid){
 
     public static PermPlayer getPlayer(UUID uuid){
         for(PermPlayer permPlayer : getPermPlayers())
-            if(permPlayer.getUuid().toString().equalsIgnoreCase(uuid.toString()))
+            if(permPlayer.getUuid().toString().replace("-","").equalsIgnoreCase(uuid.toString().replace("-",""))) {
```
Comment (guipenedo) [inline src/me/philipsnostrum/bungeepexbridge/modules/PermPlayer.java]: What's the purpose of this?

Comment (ctooley21) [reply src/me/philipsnostrum/bungeepexbridge/modules/PermPlayer.java]: zPermissions stores UUID's trimmed. So I just converted the getPlayer to that, should still work with pex and sexypex.

**Pull request merged**